### PR TITLE
Adjust error message for "U2F not supported"

### LINKED ISSUE #1816: Update "U2F Not Supported" Error Message
"Just ran across your excellent video on registering EOS tokens using Ledger Nano S - great job! That said, I've run into an error mesg on mycrypto - "Error: U2F not supported" - so could not connect. Any suggestions? Thanks!"

via https://twitter.com/JLWade4/status/997528861926715394

---

**"U2F not supported"**

could be improved to read

**"The U2F standard that hardware wallets use does not seem to be supported by your browser. Please try again using Google Chrome."**

More inspiration on the subject 😉: https://webflow.com/blog/how-not-to-write-an-error-message

### PR DESCRIPTION
**Pull request by ConnorBryan**
Closes #1816

### Description
@tayvano pointed out the verbiage for a particular U2F error was unsatisfactory. This verbiage was the error message passed along without any changes.

### Changes

* Added another check in `libs/wallet/deterministic/ledger.ts` to handle a third type of passed error
* Check that the error includes the previous message
* Create a translation entry to provide better verbiage

### Steps to Test

1. Attempt to use Ledger on Safari

### Screenshots

screen shot 2018-06-21 at 2 39 36 pm

### CODE REVIEW

**Comment by ConnorBryan** [comment]
Whoops -- I had my local branch ahead with non-changes. Fixed for future PRs, please ignore other commits.

File: common/translations/lang/en.json [hunk 1]
```diff
@@ -655,6 +655,7 @@
     "NETWORK": "Network",
     "NETWORK_2": "network",
     "PROVIDED_BY": "provided by",
-    "YOU_ARE_INTERACTING": "You are interacting with the"
+    "YOU_ARE_INTERACTING": "You are interacting with the",
```
Comment (dternyak) [inline common/translations/lang/en.json]: @wbobeirne Previously had some comments to address these strings that made their way during the ducks refactor. We don't have to do it in this PR, but just a reminder.

Comment (dternyak) [reply common/translations/lang/en.json]: https://github.com/MyCryptoHQ/MyCrypto/issues/1981 captures this

Comment (ConnorBryan) [reply common/translations/lang/en.json]: 👍

Comment (Yokymia32) [reply common/translations/lang/en.json:658]: Hey

**Pull request merged**